task/prepare.go

```diff
@@ -15,6 +15,7 @@ import (
 const (
 	minVideoBitrate         = 100_000
 	absoluteMinVideoBitrate = 5_000
+	minVideoDimensionPixels = 145
```

Comment:
Maybe we can add a very small doc here explaining where this comes from.
```suggestion
    // NVIDIA cards with Turing architecture enforce a minimum width/height of 145 pixels on transcoded videos
	minVideoDimensionPixels = 145
```